modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-unsafe-recover.adoc

```diff
@@ -0,0 +1,41 @@
+= rpk cluster partitions unsafe-recover
+
+Recover unsafely from partitions that have lost majority.
+
+IMPORTANT: This operation is unsafe because it allows you to bulk move all partitions that have lost majority when nodes are gone and irrecoverable; this may result in data loss.
```

Comment:
```suggestion
IMPORTANT: This operation is unsafe because it allows the forced leader election of the partitions that have lost majority when nodes are gone and irrecoverable; this may result in data loss.
```